src/hotspot/cpu/ppc/matcher_ppc.hpp

```diff
@@ -165,7 +165,7 @@
   static const bool supports_encode_ascii_array = true;
 
   // Returns pre-selection estimated cost of a vector operation.
-  static int vector_op_pre_select_sz_estimate(int vopc, BasicType ety, int vlen) {
+  static int op_pre_select_sz_estimate(int vopc, BasicType ety, int vlen) {
```

Comment:
For Op_Round* on PPC and S390 we currently estimate size 30 but this changes it to zero. Presumably that's correct though as those platforms don't emit any code in the corresponding match rules?

Reply:
As and when those platforms get ported, they should provide appropriate estimates. I an working on this code, though, and I'll change it back to 30 for other platforms. 
BTW, many of the numbers in that table are wrong for non-x86, and we really should provide estimates that are AArch64-realistic, but not today.